### ISSUE TITLE
A Request - Inverted Colors on Preview Pane

### ISSUE THREAD

**Issue by halyuruk** [alexandervdm/gummi#135]
Is it possible to add inverted colors option for viewing pdf files on Preview pane? 
In its current form, left pane is dark and right pane is white. It becomes hard to look when someone needs to work for several hours.

Most document viewers or pdf readers like Okular, xpdf, zathura, foxit and xreader have this option.
The setting might be added to View or Preferences section.

In addition to inverted colors option, selection of custom background colors and plain text colors would be very useful too.

Thanks

**System:**
 - Gummi version 0.6.6-4

**Comment by wltjr** [comment]
Yes please, inverted PDF to match dark themes, my eyes... :see_no_evil:
